packages/bsky/src/services/indexing/index.ts

```diff
@@ -194,7 +194,7 @@ export class IndexingService {
   async tombstoneActor(did: string) {
     this.db.assertTransaction()
     const doc = await this.didResolver.resolveDid(did)
-    if (doc.didResolutionMetadata.error === 'notFound') {
+    if (doc === null) {
```

Comment:
The key thing I'm looking for here is that the doc positively doesn't exist, versus not being able to find it due to a network error, service down, or similar.  I think you're capturing that here, just wanted to double-check!

Reply:
I found the logic for it 👍 I think I was the last to touch that— I be we can tune it up to be more accurate, but seems reasonable enough for now!

Reply:
yup yup cool 👍 

i try to return null when it doesn't exist & throw an error for a network error 👌